.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+/logs/app.log
```

I think it would be better 
```suggestion
logs/*
```